api.rst

```diff
@@ -111,11 +87,11 @@ sr            :code:`>>`                strictly right of
 nxr           :code:`&<`                does not extend to the right of, e.g. :code:`?range=nxr.(1,10)`
 nxl           :code:`&>`                does not extend to the left of
 adj           :code:`-|-`               is adjacent to, e.g. :code:`?range=adj.(1,10)`
-not           :code:`NOT`               negates another operator, see below
+and           :code:`AND`               logical :code:`AND`, see :ref:`logical_operators`
```

Comment:
As I mentioned on the above comment, AND is not really needed because it's the default. I can't find the link now, but I read somewhere that our syntax was complicated and error prone because it was thought that `and=(..)` had to be used for boolean logic.

IIRC, the `and` operator was implemented for completion, `or` was the main feature.

Perhaps we should remove it from here and just mention it on the Logical operators section.

Reply:
I disagree - it's a valid operator so we should list it here for completeness.

Maybe we can just add a hint about that default behaviour?

Reply:
> it's a valid operator so we should list it here for completeness.

Hm, yeah. Maybe we can list the `AND` as the last row on the table since its usage will be rare.

> Maybe we can just add a hint about that default behaviour?

Sounds good.